pootle/static/js/browser/components/BrowserTable.js

```diff
@@ -131,10 +131,16 @@ const BrowserTable = React.createClass({
     this.setState({ reverseSort, sortColumn });
   },
 
+  isItemDisabled(item) {
+    return item.hasOwnProperty('is_disabled') && item.is_disabled
```

Comment:
Nit: when splitting long lines, I think it'll be preferable to wrap everything in parens (we do that elsewhere AFAICT). Under some circumstance one might be tempted to wrap a long line like this:
```js
return
  imagine.this.is.a.very.very.long.line.with.a.stupid.methoCall(andMultiple, Arguments);
```
And unwillingly introduce a bug, because the function will always return `undefined` due to JS's Automatic Semicolon Insertion (ASI).